wn/_add.py

```diff
@@ -239,9 +239,9 @@ def _update_lookup_tables(
     lexicon: _AnyLexicon,
     cur: sqlite3.Cursor
 ) -> None:
-    reltypes = set(rel['relType']
+    reltypes = {rel['relType']
                    for ss in _synsets(lexicon)
-                   for rel in ss.get('relations', []))
+                   for rel in ss.get('relations', [])}
```

Comment:
By changing `set(` to `{`, it leaves the indenting of the subsequent lines at a weird spot. Maybe:

```python
    reltypes = {
        rel['relType']
        for ss in _synsets(lexicon)
        for rel in ss.get('relations', [])
    }
```

Reply:
% [black --skip-string-normalization wn/_add.py wn/_util.py wn/web.py](https://github.com/goodmami/wn/pull/189/commits/c23eb678f05f1f5e670043c20fbc376fdf92f83b)